webview/src/plots/components/comparisonTable/ComparisonTableCell.tsx

```diff
@@ -28,15 +31,25 @@ export const ComparisonTableCell: React.FC<ComparisonTableCellProps> = ({
   if (missing) {
     return (
       <div className={styles.noImageContent}>
-        <p>No Plot to Display.</p>
-        <RefreshButton
-          onClick={() =>
-            sendMessage({
-              payload: plot.revision,
-              type: MessageFromWebviewType.REFRESH_REVISION
-            })
-          }
-        />
+        {error ? (
+          <>
+            <ErrorTooltip error={error}>
+              <div>
+                <Error height={48} width={48} className={styles.errorIcon} />
```

Comment:
[F] Tried an `$error-color` `!` as well, couldn't make it work.

Reply:
sorry, there is no `!` icon but we show `!` for errors in the experiment table. What I meant was I had `!` instead of the error icon for an error but it just looked weird (unfortunately).